### PR TITLE
Add templates `python-producer` and `javascript-producer`

### PR DESCRIPTION
**Pull request by flippingbits**
## Summary of Changes

Add the new templates `python-producer` and `javascript-producer` to the ide-sidecar.

## Pull request checklist

Please check if your PR fulfills the following (if applicable):

- Tests:
    - [ ] Added new
    - [ ] Updated existing
    - [ ] Deleted existing
- [x] Have you validated this change locally against a running instance of the Quarkus dev server?
    ```shell
    make quarkus-dev
    ```
- [x] Have you validated this change against a locally running native executable?
    ```shell
    make mvn-package-native && ./target/ide-sidecar-*-runner
    ```

### CODE REVIEW

File: src/main/resources/static/templates/javascript-producer/src/producer.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+// producer should always be connected at app initialization, separately from producing message
+const connectProducer = async () => {
+  await producer.connect();
+  console.log("Connected successfully");
+};
+
+const run = async () => {
+  try {
+    let i = 0;
```
Comment (MSeal) [inline src/main/resources/static/templates/javascript-producer/src/producer.js:30]: We don't use this variable anywhere, consider removing?

File: src/main/resources/static/templates/python-producer/src/producer.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    # creates a new producer instance
+    producer = Producer(config)
+
+    # produces message until terminated
+    try:
+        while True:
+            # produces a sample message
+            key = ''.join(random.choices(string.ascii_letters + string.digits, k=5))
+            value = ''.join(random.choices(string.ascii_letters + string.digits, k=10))
```
Comment (MSeal) [inline src/main/resources/static/templates/python-producer/src/producer.py:24]: Maybe we should produce random JSON values? Fine either way just thinking about what to start with.

**Review by MSeal — APPROVED** [review]

File: src/main/resources/static/templates/javascript-producer/src/producer.js (hunk 1 above)
Comment (flippingbits) [reply src/main/resources/static/templates/javascript-producer/src/producer.js:30]: @Cerchie Mentioning you for visibility.

File: src/main/resources/static/templates/python-producer/src/producer.py (hunk 2 above)
Comment (flippingbits) [reply src/main/resources/static/templates/python-producer/src/producer.py:24]: @derek1ee Mentioning you for visibility.

**Pull request merged**